src/modules/Layout/Nav/Nav.scss

```diff
@@ -0,0 +1,20 @@
+@import 'index.scss';
+
+.j-nav {
+  display: flex; 
+  height: $hf-height-header;
+
+  &__item {
+    display: flex;
+    height: 100%;
+    align-items: center;
+    padding: 0 $hf-padding-middle;
+    color: $hf-color-white;
+    text-decoration: none;
+
+    &.active {
```

Comment:
I think you can still use BEM for this as you can use the [activeClassName](https://github.com/reactjs/react-router-tutorial/tree/master/lessons/05-active-links#active-class-name)